.changelog/16962.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+logmon: Added option for disabling logmon to task `logs` block
```

Comment:
Quicked peeked at the changelog, and we don't mention logmon much. Might be nice to frame this from the use case's perspective:

```suggestion
jobspec: Added option for disabling task log collection in the `logs` block
```

nbd though